### DISCUSSION TITLE
Reverse translation

### DISCUSSION THREAD

**Discussion by gennaios** [immersive-translate/immersive-translate#2287] [Ideas]
### 新功能 | New Feature

1. One can translate into for example one's native language. Anything -> Chinese.
2. A separate setting for reverse translation, such as Chinese pages or YouTube videos translate to something else, for example English, I think would significantly help language learning.

### 为什么您想要添加这些功能 | Why you want to add these features

1. For any pages already in the targeted language, having a setting for that to translate into something else, I think very much helps with more quickly learning a language if one wants to learn rather than only see a translation.

### 提议 | Proposal

An addition setting so that whatever is already in the target language, translate it to something else, such as one's native language.

**Comment by theowenyoung** [comment]
I do not recommend that you view immersive translate as a tool for learning English; we want to focus on simple translation actions and do not intend to introduce more special logic.
